tests/integration/fixtures/processing.py

```diff
@@ -211,7 +211,7 @@ def category_value(category):
         return 8
     if category == "transaction_indexed":
         return 9
-    if category == "feedback":
+    if category == "user_report_v2":
```

Comment:
If we want the category to be named `"feedback"` instead, we need to slap a `serde(rename="feedback")` onto the data category variant.

Reply:
If I name the category `user_report_v2` I hit the assertion error on the line below.  Feedback appears to be the correct type name.